x-pack/test/apm_api_integration/tests/alerts/alerting_api_helper.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SuperTest, Test } from 'supertest';
+import { ApmRuleType } from '@kbn/apm-plugin/common/rules/apm_rule_types';
+import { ApmRuleParamsType } from '@kbn/apm-plugin/common/rules/schema';
+import { ApmDocumentType } from '@kbn/apm-plugin/common/document_type';
+import { RollupInterval } from '@kbn/apm-plugin/common/rollup';
+import { ApmApiClient } from '../../common/config';
+
+export async function createIndexConnector({
+  supertest,
+  name,
+  indexName,
+}: {
+  supertest: SuperTest<Test>;
+  name: string;
+  indexName: string;
+}) {
+  const { body } = await supertest
+    .post(`/api/actions/connector`)
+    .set('kbn-xsrf', 'foo')
+    .send({
+      name,
+      config: {
+        index: indexName,
+        refresh: true,
+      },
+      connector_type_id: '.index',
+    });
+  return body.id as string;
+}
+
+export async function createApmRule<T extends ApmRuleType>({
+  supertest,
+  name,
+  ruleTypeId,
+  params,
+  actions = [],
+}: {
+  supertest: SuperTest<Test>;
+  ruleTypeId: T;
+  name: string;
+  params: ApmRuleParamsType[T];
+  actions?: any[];
+}) {
+  const { body } = await supertest
+    .post(`/api/alerting/rule`)
+    .set('kbn-xsrf', 'foo')
+    .send({
+      params,
+      consumer: 'apm',
+      schedule: {
+        interval: '1m',
+      },
+      tags: ['apm'],
+      name,
+      rule_type_id: ruleTypeId,
+      actions,
+    });
+  return body;
+}
+
+function getTimerange() {
+  return {
+    start: new Date(Date.now() - 24 * 60 * 60 * 1000).toISOString(),
+    end: new Date(Date.now() + 5 * 60 * 1000).toISOString(),
+  };
+}
```

Comment:
No need for a function is there?

```suggestion
const start = new Date(Date.now() - 24 * 60 * 60 * 1000).toISOString();
const end = new Date(Date.now() + 5 * 60 * 1000).toISOString();
```

Reply:
I created a function because we use the same logic to fetch both service inventory and Alerts count on Service tab